gtars/src/uniwig/mod.rs

```diff
@@ -217,18 +245,14 @@ pub fn uniwig_main(
     // Preallocate memory based on number of chromsomes from previous step
     let mut chroms: Vec<String> = Vec::with_capacity(num_chromosomes);
 
-    println!("Processing each chromosome...");
-    for chromosome in chromosomes.iter() {
+    println!("PreProcessing each chromosome...");
```

Comment:
Remove? What if we replaced this stuff with a spinner?

Reply:
Great idea. I added some simple progress bars for writing files.